CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * `env.yaml` allows to use existing keys in subsequent values
 * Fixes missing plot when generating `ploomber report` with `--backend d3`
 * Fixes error when using dotted paths in `grid` (#951)
+* Add `start_method` to Parallel executor to set the method which should be used to start child processes ([#942](https://github.com/ploomber/ploomber/issues/942))
```

Comment:
Copy the syntax from the line before. Also pull from main, 0.20 was already released.

Reply:
Also, you need to add a line on the docs folder, this was the initial issue ( we were missing documentation on this feature).

Reply:
What do you mean by adding a line on the docs folder?
I've added a description of this new argument (with a note) to `src/ploomber/executors/parallel.py` next to other similar arguments (`processes` and `print_progress`). I saw the [spec.rst](https://github.com/ploomber/ploomber/blob/master/doc/api/spec.rst#executor) but it seems like it's not the place to document new arguments.

![image](https://user-images.githubusercontent.com/103026689/183841198-429828d1-76e1-4f6b-8624-09f9f791e0e8.png)